app/src/main/java/com/github/kulebin/myfoursquareapp/adapter/VenueListAdapter.java

```diff
@@ -39,17 +46,30 @@ public int getItemCount() {
 
     public static class VenueItemViewHolder extends RecyclerView.ViewHolder implements VenueItemView {
 
+        String venueId;
+        @BindView(R.id.text_venue_name)
         TextView nameView;
+        @BindView(R.id.text_venue_location)
         TextView addressView;
+        @BindView(R.id.text_venue_rating)
         TextView ratingView;
+        @BindView(R.id.image_venue)
         ImageView imageView;
+        VenueListContract.Presentation.OnItemListener itemListener;
 
         public VenueItemViewHolder(final View itemView) {
             super(itemView);
-            this.nameView = (TextView) itemView.findViewById(R.id.venueNameTextView);
-            this.addressView = (TextView) itemView.findViewById(R.id.venueLocationTextView);
-            this.ratingView = (TextView) itemView.findViewById(R.id.venueRatingTextViewView);
-            this.imageView = (ImageView) itemView.findViewById(R.id.venueImageView);
+
+            ButterKnife.bind(this, itemView);
+            itemView.setOnClickListener(new View.OnClickListener() {
+
```

Comment:
Misha, in accordance with MVP approach it's not a good idea to initialize any Android SDK's class within a Presenter class as in such case we cannot test our Presenter by Java JUnit.